src/dlgConnectionProfiles.cpp

```diff
@@ -1787,13 +1805,23 @@ bool dlgConnectionProfiles::validateProfile()
 
         if (!check.isValid()) {
             notificationAreaIconLabelError->show();
-            notificationAreaMessageBox->setText(
-                    qsl("%1\n%2\n\n%3").arg(notificationAreaMessageBox->text(), tr("Please enter the URL or IP address of the Game server."), check.errorString()));
+            notificationAreaMessageBox->setText(QStringLiteral("%1\n%2\n\n%3").arg(notificationAreaMessageBox->text(), tr("Please enter the URL or IP address of the Game server."), check.errorString()));
```

Comment:
Why the reversion from `qsl(`...`)` back to `QStringLiteral(`...`)`?

Reply:
It seems accidental, but it's also super trivial stuff...